administrator-manual/en/subscription.rst

```diff
@@ -52,4 +52,4 @@ revert any modification to repositories and access the community ones: ::
 
 If you have installed the `new server manager` AKA `cockpit`, you can unsubscribe the machine in the `Subscription` page, clicking on **Unsubscribe** button.
 
-Refer to :ref:`software-updates-section` for more information about the community updates origin.
```

Comment:
I'd change the "Removing a subscription" section a bit more... like you did for Email. Describe the new Server Manager Unsubscribe button procedure, then the commands for the old one.

Reply:
Added sanesecurity